src/modules/cpu/kernel/brightness.hpp

```diff
@@ -187,7 +212,12 @@ RppStatus brightness_u8_u8_host_tensor(Rpp8u *srcPtr,
         // Brightness without fused output-layout toggle (NHWC -> NHWC or NCHW -> NCHW)
         else
         {
-            Rpp32u alignedLength = bufferLength & ~15;
+#if __AVX512__
+            int max_length = 64;
```

Comment:
Please remove the usage of this max_length variable and use vectorIncrementPerChannel or another suitable variable used